templates/deployments.yaml

```diff
@@ -1033,6 +1037,11 @@ spec:
         {{- if $.Values.st2sensorcontainer.postStartScript }}
         checksum/post-start-script: {{ $.Values.st2sensorcontainer.postStartScript | sha256sum }}
         {{- end }}
+        {{- if $one_sensor_per_pod }}
+        st2sensor/mode: one-sensor-per-pod
+        {{- else }}
+        st2sensor/mode: all-sensors-in-one-pod
+        {{- end }}
```

Comment:
@armab:
- Are you okay with exposing this via an annotation?
- What do you think of `st2sensor/mode`? Should this be named something else?
- What do you think about the values: `one-sensor-per-pod` and `all-sensors-in-one-pod`?

Reply:
@armab Would you prefer `stackstorm/sensor-mode` over `st2sensor/mode`?

Reply:
`stackstorm/` as a prefix/domain name looks better :+1:

Reply:
Ok I'll adjust that. Thanks for the review!